qiskit_machine_learning/algorithms/classifiers/qsvc.py

```diff
@@ -47,7 +49,12 @@ def __init__(self, *args, quantum_kernel: Optional[QuantumKernel] = None, **kwar
 
         self._quantum_kernel = quantum_kernel if quantum_kernel else QuantumKernel()
 
-        super().__init__(kernel=self._quantum_kernel.evaluate, *args, **kwargs)
+        super().__init__(
+            kernel=self._quantum_kernel.evaluate,
+            random_state=algorithm_globals.random_seed,
+            *args,
+            **kwargs,
+        )
```

Comment:
Could it be better to set `random_state` to `algorithm_globals.random_seed` only if `random_state` is not passed in `kwargs`? Now, if I want to pass my own `random_state` it is overwritten in the constructor.

Reply:
@adekusar-drl You're right, I've edited the code accordingly.